package.json

```diff
@@ -42,6 +42,7 @@
     "dev:list": "(cd packages/airnode-operation && yarn run dev:list)",
     "dev:stop": "(cd packages/airnode-operation && yarn run dev:stop)",
     "docker": "yarn docker:artifacts && yarn docker:deployer && yarn docker:node",
+    "docker:admin": "docker build --tag api3/airnode-admin:latest --file packages/airnode-admin/docker/Dockerfile .",
```

Comment:
Minor thing, but maybe `docker:build:admin` is a bit more descriptive as a script name? (And the same would apply to the other `docker build` scripts. What do you think?

Reply:
Makes it easier to add other Docker commands in the future too. e.g. `docker run ...`

Reply:
Agree.